advocacy_docs/edb-postgres-ai/ai-accelerator/installing/packages.mdx

```diff
@@ -1,75 +1,79 @@
 ---
 title: "AI Accelerator - Pipelines"
 navTitle: "Installing packages"
-description: "How to install the aidb and PGFS extensions from packages."
+description: "How to install the AI Database, PGFS, and PGvector extensions from packages."
 prevNext: true
 ---
 
 ## Installing the extension packages
 
-There are three extension packages that are required to use the AI Accelerator Pipelines:
+To use AI Accelerator Pipelines, you need three extension packages:
```

Comment:
Are we calling it AI Accelerator Pipelines or AI Accelerator - Pipelines? I prefer it without the hyphen. We'd need to change the topic title (or change this instance) depending on what we decide.

Reply:
Reading through this in the pull request, I think that's how it's referred to almost everywhere, so I will change the few instances that have the hyphen.